airgun/views/ldapauthentication.py

```diff
@@ -14,7 +14,7 @@
 
 class LDAPAuthenticationsView(BaseLoggedInView):
     title = Text(
-        "//h1[text()='LDAP authentication sources' or "
+        "//h1[text()='Authentication Sources' or text()='LDAP authentication sources' or "
         "text()='LDAP Authentication']"
```

Comment:
I wonder if this title still can be "LDAP Authentication", or when was the last time it could be. We don't really try for master to support all Satellite versions, so maybe it's time to remove that line?

Reply:
you are right this was used before 6.6, I can remove it